monorepo/website/src/pages/docs/how-to/edit-group.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: ../../../layouts/DocLayout.astro
+title: "Editing group information"
+order: 22
+---
+import CautionBox from '../../../components/asides/CautionBox.astro';
+
+Any member of a group can edit group information, which means that you can edit group information for all the groups that you are a member of.
```

Comment:
```suggestion
Any member of a group can edit group information, which means that you can edit group information for all the groups that you are a member of.

All group information except for the group's id can be edited.
```

Reply:
(minor vote against this change which IMO would slightly reduce clarity as I don't think people would expect to be able to change a numeric ID and therefore might interpret this as being the name)